### ISSUE TITLE
Set up a Python Project for Streamlit and deploy to prod

### ISSUE THREAD

**Issue by meganhicks** [department-of-veterans-affairs/abd-vro#3575]
**User Story**: 

As a VRO Platform, we need a base Python project template to kickstart development with pre-configured code quality checks and linting, ensuring that the project maintains high standards from the outset.

Acceptance Criteria:

- [x] example python project is used to create streamlit project #3552
- [x] Docker Compose: A Docker Compose file will need to be changed to expose Streamlit ports 
- [x] Integration Test Setup: Include an integration test setup within the .github/workflows directory to ensure continuous integration (CI) capabilities.
- [x] Hello World App: A simple "Hello World" application should show streamlit functionality. 
- [x] Demo to the team in Sprint Review 
- [ ] ~~Deployed to prod~~

**Comment by brostk** [comment]
The deployment is currently blocked by an LHDI issue with a pending ticket.  Websockets in LHDI do not currently function due to some side effects of an envoy filter breaking websocket connections, which are required to run Streamlit applications.  AC has been modified and another ticket will be created to reflect the deployment.  Development on future tickets (setup authentication, consume data from RDS) should not be completely blocked by this issue, but non-local testing will be.  LHDI is aware of the priority of this application.

**Comment by BerniXiongA6** [comment]
Hey @brostk when you find out what your LHDI Support ticket number is, could you leave a link to it in the comments on this ticket so we can follow it with you? Thanks so much! cc: @amylai-va @meganhicks

**Comment by brostk** [comment]
Hey @BerniXiongA6!  Here's the ticket: https://jira.devops.va.gov/browse/API-41252
